Automattic/wp-calypso
# Domains: Find similar domains on enhanced navigation

#### Changes proposed in this Pull Request

This PR updates the "Find similar domains" item in the enhanced navigation, to use the currently chosen domain as the initial search query.

#### Testing instructions

For a registered domain, click on the "Find similar domains" navigation item, and verify that the search term is correct.

Should we adjust the description/subtitle of the navigation item to something else?
